client/ladder/action-creators.ts

```diff
@@ -96,7 +189,18 @@ export function getInstantaneousSelfRank(spec: RequestHandlingSpec<void>): Thunk
   })
 }
 
-/** Navigates to the ladder standings (optionally for a given matchmaking type). */
-export function navigateToLadder(type?: MatchmakingType, transitionFn = push) {
-  transitionFn(urlPath`/ladder/${type ?? ''}`)
+/** Navigates to the ladder rankings page (optionally for a given matchmaking type and a season). */
+export function navigateToLadder(
+  matchmakingType?: MatchmakingType,
```

Comment:
You could also set a default value here instead of doing the if check below (would result in the URL not being as simple sometimes but I don't think that really matters tbh)